critiquebrainz/frontend/templates/user/reviews.html

```diff
@@ -60,6 +60,22 @@
                   {% endif %}
                   )
                 {% endif %}
+
+              {%- elif review.entity_type == 'recording' -%}
+                <strong>{{ entity.name | default(_('[Unknown recording]')) }}</strong>
+                - {{ entity['artist-credit-phrase'] | default(_('[Unknown artist]')) }}
+
+              {%- elif review.entity_type == 'artist' -%}
```

Comment:
were these items previously missing?

Reply:
Yeah, I checked the `git blame` seems so.